content/zh/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -0,0 +1,317 @@
+<!--
+---
+reviewers:
+- verb
+- yujuhong
+title: Ephemeral Containers
+content_template: templates/concept
+weight: 80
+---
+-->
+
+---
+reviewers:
+- verb
+- yujuhong
+title: 临时容器
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+<!--
+This page provides an overview of ephemeral containers: a special type of container
+that runs temporarily in an existing {{< glossary_tooltip term_id="pod" >}} to accomplish user-initiated actions such
+as troubleshooting. You use ephemeral containers to inspect services rather than
+to build applications.
+-->
+
+此页面概述了临时容器：一种特殊的容器，该容器在现有 {{< glossary_tooltip term_id="pod" >}} 中临时运行，为了完成用户启动的操作，例如故障排查。使用临时容器来检查服务，而不是构建应用程序。
+
+{{< warning >}}
+<!--
+Ephemeral containers are in early alpha state and are not suitable for production
+clusters. You should expect the feature not to work in some situations, such as
+when targeting the namespaces of a container. In accordance with the [Kubernetes
+Deprecation Policy](/docs/reference/using-api/deprecation-policy/), this alpha
+feature could change significantly in the future or be removed entirely.
+-->
```

Comment:
move line 35-41 out of the {{<warning>}} {{< /warning >}} block in case it causes parsing problem.